File: wlmedia_sample/wlmedia/src/main/java/com/ywl5320/wlmedia/listener/WlOnOutPcmDataListener.java
package com.ywl5320.wlmedia.listener;

/**
 * author : ywl5320
 * e-mail : ywl5320@163.com
 * desc   : wlmedia
 * date   : 2024/12/14
 */
public interface WlOnOutPcmDataListener {

    void onOutPcmInfo(int bit, int channel, int sampleRate);

    void onOutPcmBuffer(int size, byte[] buffers, double db);

}

File: platform/wlmedia_hm/libwlmedia/src/main/ets/com/ywl5320/wlmedia/listener/WlOnOutPcmDataListener.ts
/**
 * author : ywl5320
 * e-mail : ywl5320@163.com
 * desc   : wlmedia
 * date   : 2025/1/17
 */
export interface WlOnOutPcmDataListener {

  onOutPcmInfo(bit: number, channel: number, sampleRate: number);

  onOutPcmBuffer(size: number, buffers: Uint8Array, db: number);
}